unittest/test_ivf_nm.cpp

```diff
@@ -102,13 +102,14 @@ TEST_P(IVFNMTest, ivfnm_basic) {
     ASSERT_GT(index_->Size(), 0);
 
     LoadRawData(index_, base_dataset, conf_);
-
-    auto result = index_->GetVectorById(id_dataset, conf_);
-    AssertVec(result, base_dataset, id_dataset, nq, dim);
-
+    if (index_mode_ == knowhere::IndexMode::MODE_CPU) {
+        auto result = index_->GetVectorById(id_dataset, conf_);
+        AssertVec(result, base_dataset, id_dataset, nq, dim);
+    }
     std::vector<int64_t> ids_invalid(nq, nb);
```

Comment:
you can embrace all these lines into "if (index_mode_ == knowhere::IndexMode::MODE_CPU)“

Reply:
ok